### PR TITLE
fix: Fix layout issue on switch view when used with large text

### PR DESCRIPTION
**Pull request by nauaros**
## Summary

Fixed layout bug on the switch view within the `FormToggleItem` when used with long texts.

### CODE REVIEW

File: Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift [hunk 1]
```diff
@@ -9,7 +9,35 @@ import UIKit
 /// A view representing a switch item.
 /// :nodoc:
 public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemStyle, FormToggleItem> {
-    
+
+    private enum Layout {
+        static let switchWidth: CGFloat = 40.0
```
Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift:14]: the width is assumed, maybe its not 40 on iPad, or it will change in later iOS versions

**Review by mohammedDehairy — DISMISSED** [review]

File: Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift [hunk 2]
```diff
@@ -26,34 +51,29 @@ public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemSty
...
-    
-    // MARK: - Switch Control
-    
-    internal lazy var switchControl: UISwitch = {
-        let switchControl = UISwitch()
-        switchControl.isOn = item.value
-        switchControl.onTintColor = item.style.tintColor
-        switchControl.isAccessibilityElement = false
-        switchControl.addTarget(self, action: #selector(switchControlValueChanged), for: .valueChanged)
-        switchControl.setContentHuggingPriority(.required, for: .horizontal)
```
Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift:47]: shouldn't you keep `switchControl.setContentHuggingPriority` as well?

Comment (nauaros) [reply Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift:47]: You're right, it makes sense to keep the content hugging priority

Comment (nauaros) [reply Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift:47]: Added back

**Review by erenbesel — APPROVED** [review]

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**